logstash-core-event-java/lib/logstash/event.rb

```diff
@@ -31,7 +31,24 @@ def shutdown?; false; end;
   SHUTDOWN = ShutdownEvent.new
 end
 
+module LogStash class Event
```

Comment:
this is nitpicking but why not define the `class Event` in the previous `module LogStash` definition ?